vendor_blueprints/refresh_dataset.py

```diff
@@ -0,0 +1,128 @@
+import sys
+import argparse
+import json
+import pydomo
+import requests
+import shipyard_utils as shipyard
+
+
+EXIT_CODE_INVALID_CREDENTIALS = 200
+EXIT_CODE_DATASET_NOT_FOUND = 201
+EXIT_CODE_REFRESH_ERROR = 202
+
+
+def get_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--client-id', dest='client_id', required=True)
+    parser.add_argument('--secret-key', dest='secret_key', required=True)
+    parser.add_argument('--email', dest='email', required=True)
+    parser.add_argument('--password', dest='password', required=True)
+    parser.add_argument('--domo-instance', dest='domo_instance', required=True)
+    parser.add_argument('--dataset-id', dest='dataset_id', required=True)
+    args = parser.parse_args()
+    return args
+
+
+def get_access_token(email, password, domo_instance):
+    """
+    Generate Access Token for use with internal content APIs.
+
+    email (str): email address of the user on domo.com
+    password (str): login password of the user domo.com
+    """
+    auth_api = f"https://{domo_instance}.domo.com/api/content/v2/authentication"
+
+    auth_body = json.dumps({
+        "method": "password",
+        "emailAddress": email,
+        "password": password
+    })
+
+    auth_headers = {'Content-Type' : 'application/json'}
+    auth_response = requests.post(auth_api, data=auth_body, 
+                               headers=auth_headers)
+
+    if auth_response.json()["success"] is False: # Failed to login
```

Comment:
Check out the download script to see how I changed this. If the domo instance is provided incorrectly, this section fails because the 403 response fields differ.

Reply:
Should also have a separate error for "invalid domo instance" or "invalid account" which is what I think I used. Error code should be the same as the export_card_to_file